modules/ts-capi-cli.adoc

```diff
@@ -5,11 +5,36 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="ts-capi-cli_{context}"]
-= Returning the intended machines when using the CLI
+= Referencing the intended objects when using the CLI
 
-For clusters that use the Cluster API, `oc` commands such as `oc get machine` return results for Cluster API machines.
-Because the letter `c` precedes the letter `m` alphabetically, Cluster API machines appear in the returned results before Machine API machines do.
-You can specify the type of machines to return by using the corresponding fully qualified name.
+For clusters that use the Cluster API, {oc-first} commands prioritize Cluster API objects over Machine API objects.
+
+This behavior impacts any `oc` command that acts upon any object that is represented in both the Cluster API and the Machine API.
+This explanation uses the `oc get machine` command, which lists the machines in a cluster, as an example.
+
+[discrete]
+=== Cause
+
+When you run an `oc` command, `oc` communicates with the Kube API server to determine which objects to act upon.
+The Kube API server uses the first installed custom resource definition (CRD) it encounters alphabetically when an `oc` command is run.
+
+CRDs for Cluster API objects are in the `cluster.x-k8s.io` group, while CRDs for Machine API objects are in the `openshift.io` group.
```

Comment:
`machine.openshift.io`; my examples in Slack were incomplete. Looks like all the commands listed in the other file are correct, though.